### DISCUSSION TITLE
Public roadmap... thank you so much!!!

### DISCUSSION THREAD

**Discussion by AaronBeaudoin** [directus/directus#14248] [Feature Requests]
I cannot express how amazing it is to be able to be able to **see what the Directus team is planning** on via a public roadmap. The fact that it is subject to change doesn't matter much, since we're used to many organizations providing little to no information at all, even for widely used software that lots of people depend on. The Directus approach is so **refreshing**. 

The ability to be able to sponsor a feature to give it a timeline and help get it developed is also an absolutely fantastic idea. I wish every organization did that. I've also noticed an incredible level of communication from the Directus team on GitHub. I've personally been impressed at how much the team seems to care about thoughts and ideas from the community.

<hr>

**I just want to say:** Thank you for being transparent about your future goals and connecting with the community, and keep doing it! I think Directus has the potential to someday fill many roles and functions of even complex internal enterprise systems. As the community continues to grow, I think love for Directus will grow as well. I'm sure many people, including myself, are watching with great excitement and can't wait to see (and help) Directus reach greater success. Thanks so much for your hard work!!

**Comment by abdonrd** [comment]
I don't know if it is 100% up to date, but there is this page: https://directus.io/road-map/

**Reply by abdonrd** [reply]
@rijkvanzanten a few things to update in the roadmap page:

- The `Request a feature` buttons have an outdated link:
https://github.com/directus/directus/discussions/new?category=ideas
Probably should be: https://github.com/directus/directus/discussions/new?category=feature-requests
- The `GitHub` link in the Current & Completed sections is outdated:
https://github.com/directus/directus/discussions/categories/implemented
Probably should be: https://github.com/directus/directus/discussions?discussions_q=is%3Aopen+label%3AImplemented

**Reply by rijkvanzanten** [reply]
Thanks for the heads up! We've updated those links 👍🏻

**Reply by abdonrd** [reply]
Sure!
